controllers/update_flow_steps.go

```diff
@@ -0,0 +1,312 @@
+package controllers
```

Comment:
Let's maybe split this file into two, one with "flow-framework" code and other with "ytsaurus update flow" code for better readability for our flow building code.

Reply:
I've moved couple of methods to the sync.go.